### PR TITLE
Added an NS430 FIX Format for importing Nav Waypoints to said device.

### PR DESCRIPTION
**Pull request by Wamwod**
Easily input waypoints into the NS430 using the navaids.dat file useful for making flight plans for AC without advanced navigation systems.

### CODE REVIEW

**Review by rkusa — CHANGES_REQUESTED** [review]
Cool idea, thanks for the addition!

**Comment by rkusa** [comment]
Random thought, where is the `navaids.dat` saved? Because in theorie, Scratchpad could also directly write to it (as a possible future addition).

**Comment by Wamwod** [comment]
> Random thought, where is the `navaids.dat` saved? Because in theorie, Scratchpad could also directly write to it (as a possible future addition).

It's stored in the main dcs folder under .\DCS World OpenBeta\Mods\aircraft\NS430\Cockpit\Scripts\avionics\terrain
I'll work on the changes and enable the button for NS430 hopefully before 0400Z

File: Scripts/Hooks/scratchpad-hook.lua [hunk 1]
```diff
@@ -287,29 +292,39 @@ local function loadScratchpad()
             return {DDM = {lonDegreesWidth = 3}, MGRS = true}
         elseif ac == "AH-64D_BLK_II" then
             return {DDM = {precision = 2, lonDegreesWidth = 3}, MGRS = true}
+        elseif ac == "UH-1H" then
+            return {DDM = true, DMS = true}
```
[the remark this answers is not in the document]
Comment (Wamwod) [reply Scripts/Hooks/scratchpad-hook.lua]: I went with (2) committed and should be ready to merge LMK if there's anything else I might have overlooked.

**Comment by rkusa** [comment]
Thanks again for the PR. I've resolved the conflicts and merged the PR as 255164f1d688e15dd73ca904e48a57b9a42dd3cc.

**Pull request closed without merging**